Themes/default/css/index.css

```diff
@@ -3630,6 +3630,9 @@ h3.catbg .main_icons::before, h3.catbg .icon {
 .cat_bar + .title_bar {
 	margin-top: 0;
 }
+.cat_bar:not(:first-of-type) {
+	margin: 10px 0 0 0;
+}
```

Comment:
this doesn't seem to have any effect on the the agreements page, instead it just adds extra space everywhere else, I think it should be reverted

Reply:
oh but it does
![Untitled](https://user-images.githubusercontent.com/431799/91364242-268a9500-e7b3-11ea-893d-b9ee9a61a6f4.png)

Reply:
no it doesn't, because the element above it has bottom margin, just try and remove these rules and you'll see

Reply:
huh, so it dues. SMF's highly complicated CSS is very confusing to me. At least in 2.0's CSS file I could find my way around!